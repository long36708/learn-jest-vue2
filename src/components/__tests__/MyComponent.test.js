/**
 * @Author: longmo
 * @Date: 2025-01-19 14:43:59
 * @LastEditTime: 2025-01-19 14:45:07
 * @FilePath: src/components/__tests__/MyComponent.test.js
 * @Description:
 */
import { mount } from "@vue/test-utils";
import Component from "../MyComponent.vue";
test("renders correctly", () => {
  const wrapper = mount(Component);
  expect(wrapper.element).toMatchSnapshot();
});
