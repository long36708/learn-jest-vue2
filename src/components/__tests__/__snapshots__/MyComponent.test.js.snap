// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    10
  </div>
   
  <button>
    增加
  </button>
</div>
`;
